src/Exception/Bucket/BucketAlreadyExists.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Storj\Uplink\Exception\Bucket;
+
+class BucketAlreadyExists extends BucketException
```

Comment:
Shouldn't all exception names end with `Exception`?

Reply:
That's a stylistic opinion, I find it rather verbose.